src/Microsoft.OData.Client/ALinq/DataServiceQueryProvider.cs

```diff
@@ -248,9 +250,10 @@ private TElement ParseAggregateSingletonResult<TElement>(QueryResult queryResult
                     {
                         case ODataReaderState.ResourceEnd:
                             entry = reader.Item as ODataResource;
-                            if (entry != null && entry.Properties.Any())
+                            IEnumerable<ODataProperty> properties = entry.Properties.OfType<ODataProperty>();

```

Comment:
Can properties ever be null ?

Reply:
@wachugamaina Not in this particular case